### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by jeffreyclu**
# What Changed

https://github.com/intuit/accessibility-snippets/issues/40

- Added `jest` testing framework
- Renamed `css-snippets.code-snippets` to `css-snippets.json`
- Renamed `react-snippets.code-snippets` to `react-snippets.json`
- Added unit tests for both snippet files

# Why

The `.code-snippets` files had to be renamed as `.json` files so that they could be imported and tested in the test file.

Todo:
- [ ] Add Semantic Version Label 
- [ ] Add tests
- [ ] Add docs

### CODE REVIEW

File: snippets/react-snippets.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "DialogAria": {
         "prefix": [
-            "DialogAria",
```
Comment (jeffreyclu) [inline snippets/react-snippets.json:4]: had to remove all trailing commas and inline comments

**Comment by jeffreyclu** [comment]
A couple of follow up comments:

- Since the files were renamed, it will probably break the code in other places.
- I followed this very helpful example: https://github.com/andys8/vscode-jest-snippets/blob/master/snippets/snippets.test.js
- Currently the react-snippets aren't sorted alphabetically, and I added a test to account for this, but I think sorting the snippets file is out of scope for this PR
- This can probably be added as a CI step

File: __tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const allSnippets = [cssSnippets, reactSnippets];
+
+const unique = (xs) => [...new Set(xs)];
+
+describe.each(allSnippets)("suite $#", (snippet) => {
+    it("has entries", () => {
+        expect(Object.keys(snippet).length).toBeGreaterThan(0);
+    });
+
+    // TODO: get this test working
```
Comment (kendallgassner) [inline __tests__/index.js:15]: you should add an issue for this 🤠

**Review by kendallgassner — APPROVED** [review]

**Review by kendallgassner — APPROVED** [review]

**Comment by syedareehaquasar** [comment]
Hey! @kendallgassner , @jeffreyclu ! I am not able to verify tests working... I am getting errors in the modules imported as .json not found and when I change it to actual file names it gives me an error in syntax... Can you help me with how to verify these tests?

**Comment by syedareehaquasar** [comment]
I actually needed there to work before working on automating PR tests...

**Pull request merged**